packages/@aws-cdk/aws-ecs/lib/firelens-log-router.ts

```diff
@@ -109,14 +110,22 @@ export interface FirelensLogRouterDefinitionOptions extends ContainerDefinitionO
 function renderFirelensConfig(firelensConfig: FirelensConfig): CfnTaskDefinition.FirelensConfigurationProperty {
   if (!firelensConfig.options) {
     return { type: firelensConfig.type };
+  } else if (firelensConfig.options.configFileValue === undefined) {
+    // config file options must be set together and are optional
+    return {
+      type: firelensConfig.type,
+      options: {
+        'enable-ecs-log-metadata': firelensConfig.options.enableECSLogMetadata ? 'true' : 'false',
```

Comment:
```suggestion
        'enable-ecs-log-metadata': firelensConfig.options.enableECSLogMetadata ?? 'false',
```

Reply:
can you change line 126 as well. i think it makes more sense this way since `enableECSLogMetadata` is a boolean already.

Reply:
So the value we are rendering is a string and right now with your suggestion its failing with:

```
@aws-cdk/aws-ecs: lib/firelens-log-router.ts:121:9 - error TS2322: Type 'boolean | "false"' is not assignable to type 'string'.
@aws-cdk/aws-ecs:   Type 'false' is not assignable to type 'string'.
@aws-cdk/aws-ecs: 121         'enable-ecs-log-metadata': firelensConfig.options.enableECSLogMetadata ?? 'false',
```